### PR TITLE
Upgrade tangram-es and podspec to 0.6.0

### PR DESCRIPTION
**Pull request by matteblair**
https://github.com/tangrams/tangram-es/releases/tag/0.6.0

_Release notes incoming_

### CODE REVIEW

**Review by msmollin — APPROVED** [review]

**Comment by karimnaaji** [comment]
@matteblair The size increase a little bit more in here (57 on master vs 69 with the new one). It seems that the only new library we added from 0.5.1 to 0.6.0 is the float to string parsing, I don't know how this library is set up but I wonder if we need all of it or if we could cherry-pick what we actually use in Tangram.

**Comment by matteblair** [comment]
You're right, it does seem to be bigger. I'm not sure what would account for that, since `double-conversion.a` is only 400kb in release mode. 🤔

**Comment by karimnaaji** [comment]
Looking at the history 0.5.0 was 65 (https://github.com/tangrams/ios-framework/blob/27690fd5934bbb8c53cf9552cda301a1ea44039c/TangramMap.framework/TangramMap). Something odd must have happened when generating 0.5.1, we might want to use that oddity if it reduces the size like that.

**Review by karimnaaji — APPROVED** [review]

File: Tangram-es.podspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name    = 'Tangram-es'
-  s.version = '0.5.1'
+  s.version = '0.6.0'
```
Comment (karimnaaji) [inline Tangram-es.podspec:4]: Just updated the docs on tangram-docs, `s.documentation_url` can be updated to point to `https://mapzen.com/documentation/tangram/ios-framework/0.6.0/`. Let's hold on merging until https://github.com/tangrams/tangram-docs/pull/208 is in.

**Comment by msmollin** [comment]
tangrams/tangram-docs#208 merged in so went ahead and merged this.

**Pull request merged**